python2/plan.sh

```diff
@@ -1,39 +1,68 @@
 pkg_name=python2
-pkg_version=2.7.12
+pkg_distname=Python
+pkg_version=2.7.13
 pkg_origin=core
 pkg_license=('Python-2.0')
 pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
 pkg_description="Python is a programming language that lets you work quickly
   and integrate systems more effectively."
 pkg_upstream_url=https://www.python.org/
-pkg_dirname=Python-${pkg_version}
+pkg_dirname=${pkg_distname}-${pkg_version}
 pkg_source=https://www.python.org/ftp/python/${pkg_version}/${pkg_dirname}.tgz
-pkg_filename=${pkg_dirname}.tgz
-pkg_shasum=3cb522d17463dfa69a155ab18cffa399b358c966c0363d6c8b5b3bf1384da4b6
-pkg_deps=(core/glibc core/gcc-libs core/coreutils core/make core/ncurses core/zlib core/readline core/openssl core/bzip2 core/sqlite)
-pkg_build_deps=(core/linux-headers core/gcc)
+pkg_shasum=a4f05a0720ce0fd92626f0278b6b433eee9a6173ddf2bced7957dfb599a5ece1
+pkg_deps=(
+  core/bzip2
+  core/gcc-libs
+  core/gdbm
+  core/glibc
+  core/ncurses
+  core/openssl
+  core/readline
+  core/sqlite
+  core/zlib
+)
+pkg_build_deps=(
+  core/coreutils
+  core/diffutils
+  core/gcc
+  core/gdb
+  core/linux-headers
+  core/make
+  core/util-linux
+)
 pkg_lib_dirs=(lib)
 pkg_bin_dirs=(bin)
-pkg_include_dirs=(include Include)
+pkg_include_dirs=(include)
 pkg_interpreters=(bin/python bin/python2 bin/python2.7)
 
 do_prepare() {
-    sed -i.bak 's/#zlib/zlib/' Modules/Setup.dist
-    sed -i -re "/(SSL=|_ssl|-DUSE_SSL|-lssl).*/ s|^#||" Modules/Setup.dist
+  sed -i.bak 's/#zlib/zlib/' Modules/Setup.dist
+  sed -i -re "/(SSL=|_ssl|-DUSE_SSL|-lssl).*/ s|^#||" Modules/Setup.dist
 
-    # Allow embedded sqlite to load extensions
-    # Uncertain if this is absolutely required, leaving commented for now.
-    # https://docs.python.org/2/library/sqlite3.html#f1
-    # sed -i 's/sqlite_defines.append(("SQLITE_OMIT_LOAD_EXTENSION", "1"))//g' setup.py
+  # Allow embedded sqlite to load extensions
+  # Uncertain if this is absolutely required, leaving commented for now.
+  # https://docs.python.org/2/library/sqlite3.html#f1
+  # sed -i 's/sqlite_defines.append(("SQLITE_OMIT_LOAD_EXTENSION", "1"))//g' setup.py
 }
 
 do_build() {
-    ./configure --prefix="${pkg_prefix}" \
+    ./configure --prefix="$pkg_prefix" \
                 --enable-shared \
+                --enable-unicode=ucs4 \
```

Comment:
This comes from [`python:2-alpine`](https://github.com/docker-library/python/blob/master/2.7/alpine/Dockerfile#L60)